notebooks/official/generative_ai/text_embedding_api_cloud_next_new_models.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>Does it "cloud_next" in the file name "text_embedding_api_cloud_next_new_models.ipynb" and Cloud Next in the title refer to Google Cloud Next? If so, we should consider using a more descriptive name, and avoid using "cloud_next" and "new".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>